scripts/stanford-filter.script

```diff
@@ -108,6 +108,84 @@
     +(ManufacturerModelName.containsIgnoreCase("Sensation"))
    )
   )
+
+   // Siemens SOMATOM Definition Edge, SOMATOM DEFINITION AS+, SOMATOM FORCE
+  +(
+    Manufacturer.startsWithIgnoreCase("SIEMENS") 
+    *
+    (
+        Modality.equalsIgnoreCase("CT") 
+        + 
+        Modality.equals("PT")
+    )
+    *
+    (
+        ImageType.contains("DERIVED")
+    )
+    *
+    (
+        (
+            (
+                ManufacturerModelName.containsIgnoreCase("SOMATOM Definition Edge") 
+                + 
+                ManufacturerModelName.containsIgnoreCase("SOMATOM FORCE")
+            ) 
+            * 
+            [0018,1020].containsIgnoreCase("syngo CT VB20A")
+        )
+    +
+    (
+        ManufacturerModelName.containsIgnoreCase("SOMATOM DEFINITION AS+")
+        * 
+        (
+            [0018,1020].containsIgnoreCase("syngo CT VB20A") 
+            + 
+            [0018,1020].containsIgnoreCase("syngo CT VA48A")
+        )
+    )
+   )
+   //*( The row and col are very different everytime, at least row is 512 or col is 512 from all the samples I looked at.
+   //   (Rows.equals("679") * Columns.equals("512"))
+   //   +(Rows.equals("512") * (Columns.equals("512") + Columns.equals("547") + Columns.equals("605") + Columns.equals("630") + Columns.equals("632") + Columns.equals("682") + Columns.equals("708")))
+   //   +(Rows.equals("665") * Columns.equals("512"))
+   //)
+   )
+
+   // Siemens SOMATOM DEFINITION FLASH, version SYNGO CT VB20A
+  +(Manufacturer.startsWithIgnoreCase("SIEMENS") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
+   *(ImageType.contains("DERIVED"))
+   *ManufacturerModelName.containsIgnoreCase("SOMATOM DEFINITION FLASH") 
+   *[0018,1020].containsIgnoreCase("syngo CT VB20A")
+   *(
+    [0018,1018].containsIgnoreCase("Volume Viewer")
+    +
+    [0018,1018].containsIgnoreCase("")
```

Comment:
When I tried this in rule, it didn't filter in the series as my series didn't have 0018,1018 tag and hence it didn't match on this part.
Let's make sure this rule matches to the series we are testing